pkg/transform/oauth/github_test.go

```diff
@@ -1,20 +1,48 @@
 package oauth_test
 
 import (
+	"encoding/json"
 	"io/ioutil"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/fusor/cpma/pkg/ocp3"
-	"github.com/fusor/cpma/pkg/ocp4/oauth"
+	"github.com/fusor/cpma/pkg/transform/oauth"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	configv1 "github.com/openshift/api/legacyconfig/v1"
+	k8sjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
 )
 
 func TestTransformMasterConfigGithub(t *testing.T) {
 	file := "testdata/github-test-master-config.yaml"
 	content, _ := ioutil.ReadFile(file)
-	masterV3 := ocp3.MasterDecode(content)
+
+	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
+	var masterV3 configv1.MasterConfig
+	_, _, _ = serializer.Decode(content, nil, &masterV3)
```

Comment:
Are we eating the error here? Any reason to do that?

Reply:
@dymurray I will take care of error handling in tests here https://github.com/fusor/cpma/pull/112 after https://github.com/fusor/cpma/pull/128 gets merged.

Reply:
Awesome, thanks!